include/dem/particle_wall_nonlinear_force.h

```diff
@@ -84,40 +88,66 @@ class ParticleWallNonLinearForce
 
 private:
   /**
-   * @brief No rolling resistance torque model
+   * @brief No rolling resistance torque model.
    *
-   * @param particle_one_properties Particle one properties
-   * @param particle_two_properties Particle two properties
-   * @param effective_rolling_friction_coefficient Effective rolling friction coefficient
-   * @param normal_force_norm Normal force norm
+   * @param[in] particle_properties Particle one properties.
+   * @param[in] effective_rolling_friction_coefficient Effective rolling
+   * friction coefficient.
+   * @param[in] effective_rolling_viscous_damping_coefficient Effective rolling
+   * viscous damping coefficient.
+   * @param[in] normal_force_norm Normal force norm.
+   * @param[in] dt DEM time step.
+   * @param[in] normal_spring_constant Normal spring constant.
+   * @param[in] normal_unit_vector Normal unit contact vector between the
+   * particle and the wall.
+   * @param[in,out] cumulative_rolling_resistance_spring_torque cumulative
+   * rolling resistance torque for the EPSD rolling resistance model.
    *
-   * @return rolling resistance torque
+   * @return Rolling resistance torque equal to the null vector.
```

Comment:
```suggestion
   * @return Rolling resistance torque equal to a zero vector.
```

I think you should use the word tensor here instead of vector but I understand we started by vector so maybe later.